gilbreth-e009.rcac.purdue.edu
0
The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) xalt/1.1.2
/home/ahmedb/.conda/envs/cent7/2020.11-py38/wav2letter/lib/python3.8/site-packages/torch/_tensor.py:575: UserWarning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (Triggered internally at  /opt/conda/conda-bld/pytorch_1623448278899/work/aten/src/ATen/native/BinaryOps.cpp:467.)
  return torch.floor_divide(self, other)
Creating regression obj for: 'speech2text'
Loading ANN features at bin-width: 20
It takes 750.47 seconds to load features...!
Working with '180728'
# of iterations requested: 5, 
# of lambda samples per iteration: 20
Itr: 1:
K_fold for session: 180728
Itr: 2:
K_fold for session: 180728
Itr: 3:
K_fold for session: 180728
Itr: 4:
K_fold for session: 180728
Itr: 5:
K_fold for session: 180728
It takes (on avg.) 143.47 sec (all lmbdas). (time for 5-folds)
It takes (on avg.) 5.66 sec/mapping.
It takes (on avg.) 2.49 minutes/iteration...!
Data saved for model: 'speech2text', session: '180728',    bin-width: 20ms, delay: 0ms at file: '/depot/jgmakin/data/auditory_cortex/correlation_results/cross_validated_correlations/speech2text_corr_results.csv'
Working with '180717'
# of iterations requested: 5, 
# of lambda samples per iteration: 20
Itr: 1:
K_fold for session: 180717
Itr: 2:
K_fold for session: 180717
Itr: 3:
K_fold for session: 180717
Itr: 4:
K_fold for session: 180717
Itr: 5:
K_fold for session: 180717
It takes (on avg.) 144.38 sec (all lmbdas). (time for 5-folds)
It takes (on avg.) 5.60 sec/mapping.
It takes (on avg.) 2.50 minutes/iteration...!
Data saved for model: 'speech2text', session: '180717',    bin-width: 20ms, delay: 0ms at file: '/depot/jgmakin/data/auditory_cortex/correlation_results/cross_validated_correlations/speech2text_corr_results.csv'
Working with '180627'
# of iterations requested: 5, 
# of lambda samples per iteration: 20
Itr: 1:
K_fold for session: 180627
Itr: 2:
K_fold for session: 180627
Itr: 3:
K_fold for session: 180627
Traceback (most recent call last):
  File "run_regression.py", line 96, in <module>
    corr_dict = obj.cross_validated_regression(session, bin_width=bin_width, delay=delay,
  File "/home/ahmedb/projects/Wav2Letter/Auditory_Cortex/auditory_cortex/regression.py", line 540, in cross_validated_regression
    lmbda_loss = self.k_fold_CV(session, mapping_set=mapping_set, lmbdas=lmbdas, k=k)
  File "/home/ahmedb/projects/Wav2Letter/Auditory_Cortex/auditory_cortex/regression.py", line 632, in k_fold_CV
    Beta = utils.reg(train_x, train_y, lmbda)
  File "/home/ahmedb/projects/Wav2Letter/Auditory_Cortex/auditory_cortex/utils.py", line 308, in reg
    b = module.matmul(X.transpose((0,2,1)), y)
  File "/home/ahmedb/.conda/envs/cent7/2020.11-py38/wav2letter/lib/python3.8/site-packages/cupy/_core/_gufuncs.py", line 684, in __call__
    self._apply_func_to_inputs(
  File "/home/ahmedb/.conda/envs/cent7/2020.11-py38/wav2letter/lib/python3.8/site-packages/cupy/_core/_gufuncs.py", line 399, in _apply_func_to_inputs
    func(*args, out=outs)
  File "cupy/_core/_routines_linalg.pyx", line 705, in cupy._core._routines_linalg.matmul
  File "cupy/_core/_routines_linalg.pyx", line 786, in cupy._core._routines_linalg.matmul
  File "cupy/_core/core.pyx", line 2592, in cupy._core.core.ascontiguousarray
  File "cupy/_core/core.pyx", line 171, in cupy._core.core.ndarray.__init__
  File "cupy/cuda/memory.pyx", line 698, in cupy.cuda.memory.alloc
  File "cupy/cuda/memory.pyx", line 1375, in cupy.cuda.memory.MemoryPool.malloc
  File "cupy/cuda/memory.pyx", line 1396, in cupy.cuda.memory.MemoryPool.malloc
  File "cupy/cuda/memory.pyx", line 1076, in cupy.cuda.memory.SingleDeviceMemoryPool.malloc
  File "cupy/cuda/memory.pyx", line 1097, in cupy.cuda.memory.SingleDeviceMemoryPool._malloc
  File "cupy/cuda/memory.pyx", line 1335, in cupy.cuda.memory.SingleDeviceMemoryPool._try_malloc
cupy.cuda.memory.OutOfMemoryError: Out of memory allocating 3,356,803,072 bytes (allocated so far: 14,219,946,496 bytes).
